src/actions/attack.cpp

```diff
@@ -1585,7 +1585,7 @@ void attack_unit_and_advance(const map_location& attacker,
 int under_leadership(const unit &u, const map_location& loc, const_attack_ptr weapon, const_attack_ptr opp_weapon)
 {
 	unit_ability_list abil = u.get_abilities_weapons("leadership", loc, weapon, opp_weapon);
-	unit_abilities::effect leader_effect(abil, 0, false);
+	unit_abilities::effect leader_effect(abil, 0, false, const_attack_ptr() , true);
```

Comment:
Why should is_cumulable be true for leadership, but false for any other ability?

Reply:
cumulative don't work to same for leadership and other abilities like chance_to_hit or cumulative=yes using a refence base_value like valueif ref>value, then with leadership ref ever 0 but cumulative=yes is used for add value of cumulative leadership to value to other leadership, is_cumulable vriable used for disctinct leadership abilities to other in effect